distribution/docs/pom.xml

```diff
@@ -119,6 +121,42 @@
     </profiles>
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.codehaus.gmaven</groupId>
+                <artifactId>groovy-maven-plugin</artifactId>
+                <version>2.1</version>
+                <inherited>false</inherited>
+                <executions>
+                    <execution>
+                        <id>aggregate-deps</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>execute</goal>
+                        </goals>
+                        <configuration>
+                            <source>
+                                def lines = []
+                                new File('${highest-basedir}').traverse(type: groovy.io.FileType.FILES, nameFilter: ~/maven-deps.txt/) { file ->
+                                    for (line in file.readLines())
+                                        lines.add(line.trim())
+                                }
+                                def output = ""
+                                Outer:
```

Comment:
❓ you're killing me boss...labels!?

Reply:
don't look at the standard libraries - you might feint